modules/google-cloud/README.adoc

```diff
@@ -0,0 +1,61 @@
+# Google Cloud Module
+
+Within our XTDB node, we can make use of Google Cloud Services for certain purposes. Currently, we can:
+
+* Use *Google Cloud Storage* as the XTDB Object Store.
+
+### Project Dependency 
+
+In order to use any of the Azure services, you will need to include a dependency on the xtdb.azure module.
```

Comment:
did you copy paste this?! :scream: 

(j/k, obv, but looks like this file needs a find/replace)

Reply:
So close.... I guess my find and replace missed a couple :scream: